src/test/resources/TestNG.xml

```diff
@@ -1,14 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
-<suite name="TestNG">
+<suite name="TestNG" parallel="methods" thread-count="1">
     <parameter name="BaseURL" value="https://bbb.testpro.io/" />
 
     <test name="Example Test" preserve-order="false">
         <classes>
-            <class name="Homework17" />
-<!--            <class name="ProfileTests"/>-->
-<!--            <class name="BaseTest"/>-->
+           <!-- <class name="LoginTests"/>-->
+            <class name="Homework25"/>
+            <class name="BaseTest"/>
+     <!--       <class name="Homework17A"/> -->
+       <!--     <class name="Homework18"/> -->
+   <!--         <class name="Homework19"/>
+            <class name="Homework20"/>
+            <class name="Homework21"/>
+            <class name="Homework22"/>-->
+    <!--        <class name="ProfileTests"/>-->
+
+     <!--       <class name = "LoginTests"></class> -->
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.